task_processing/plugins/kubernetes/task_config.py

```diff
@@ -32,7 +32,8 @@
 # or (max filename length) - (max namespace length) -(pod UID length) - (separators)
 # but let's give ourselves a little buffer so we'll round down a bit
 MAX_POD_NAME_LENGTH = 150
-VALID_POD_NAME_REGEX = '[a-z0-9]([.-a-z0-9]*[a-z0-9])?'
+MAX_DNS_SUBDOMAIN_NAME_LENGTH = 253
+VALID_DNS_SUBDOMAIN_NAME_REGEX = '[a-z0-9]([.-a-z0-9]*[a-z0-9])?'
```

Comment:
I think this might need ^ and $ to actually test the whole name, or we might want to use re.fullmatch?